### PR TITLE
Docs/2834/update primary readmes

### PR DESCRIPTION
**Pull request by knoffi**
# Primary Readme-files need updates

Issue: #2834

## Description

The Readme's of visualization, analysis and project root need an update. They were checked on Windows and macOS, useful notes were added and a text style approved by the PO was applied.

### CODE REVIEW

**Comment by phanlezz** [comment]
Visualization readme will be updated, but the more general question "what should be in gh-pages and what should be in readme"  is for now moved to the backlog

File: visualization/README.md [hunk 1]
```diff
@@ -7,27 +7,44 @@
...
+-   Run an operating system specific [standalone](#run-a-standalone)
+-   Build it yourself from the github [repository](#build-it-yourself)
+-   Try out the [online version](https://maibornwolff.github.io/codecharta/visualization/app/index.html?file=codecharta.cc.json&file=codecharta_analysis.cc.json)
 
--   install with `npm install codecharta-visualization -g`
--   run with `codecharta-visualization`
+## NPM Package
 
-#### Install application for desktop usage
+> Make sure you have [Node.js](https://nodejs.org/en/download) installed <br>
```
Comment (Hall-Ma) [inline visualization/README.md:26]: I would add the min version we need to install.

File: visualization/README.md [hunk 2]
```diff
@@ -7,27 +7,44 @@
...
+> For reference: [Codecharta-Visualization on NPM](https://www.npmjs.com/package/codecharta-visualization)
 
--   Download or build [latest version](https://github.com/MaibornWolff/codecharta/releases/latest) for your system.
--   Doubleclick the system specific Runnable. You may be required to give it executable rights.
+```bash
+# Install the package globally via npm
+$ npm i -g codecharta-visualization
+# Run it anywhere (you might need administrative rights/sudo)
+$ codecharta-visualization
+# You might need to restart your terminal
```
Comment (Hall-Ma) [inline visualization/README.md:34]:
```suggestion
# Run it anywhere (you might need administrative rights/sudo)
# You might need to restart your terminal
$ codecharta-visualization
```

**Review by Hall-Ma — APPROVED** [review]
Very good! 👍

File: visualization/README.md (hunk 1 above)
Comment (phanlezz) [reply visualization/README.md]: Thanks will do that 👍

File: visualization/README.md (hunk 2 above)
Comment (phanlezz) [reply visualization/README.md]: Yes, thats bette! 👍

**Pull request merged**